### ISSUE TITLE
feat: design oss-friends page

### ISSUE THREAD

**Issue by shreya-gr** [firecamp-dev/website#7]
Firecamp is now entering to the group of oss friends. 

To make it work, we need to design the new page for oss friends similar to this - https://formbricks.com/oss-friends

**Task** 

- [x] Design new page with similar UI of tabs. Reference - https://formbricks.com/oss-friends
- [x] url will be https://firecamp.io/oss-friends
- [x] use formbrick's api to show data - https://github.com/formbricks/formbricks/blob/main/apps/formbricks-com/pages/oss-friends.tsx
- [x] add the oss-friends page link to footer of Firecamp website in section of  **community** > below discord server. Label : 'OSS Friends'

**Comment by Siddhant-K-code** [comment]
> @shreya-gr: hey @Siddhant-K-code are you interested in working on this issue?

Ahh! I am quite occupied right now. I will find some time on the weekends, let me share this issue with socials also. We might find a new contributor.

**Comment by jatingodnani** [comment]
@shreya-gr interested

**Comment by rohitmondal03** [comment]
@shreya-gr can you assign me this issue ?

**Comment by rohitmondal03** [comment]
@shreya-gr the given API in not able to fetch data.
![Screenshot (109)](https://github.com/firecamp-dev/website/assets/107113353/f9ff2441-a840-4734-a0f8-e7b353ddaef7)

I have also tried fetching from this API in vanilla JS file, but it is giving same error !

**Comment by Nishchit14** [comment]
@rohitmondal03 It's a CORS issue. Once the page is ready, we need to submit it to get access to this api from our domain firecamp.io. meanwhile you can follow these steps

1. open this api in the browser https://formbricks.com/api/oss-friends
2. copy the response in JSON
3. use mock promise with this JSON instead of calling the actual API
4. and start building the page
5. add an `OSS Friend` link in the footer section

Once the API access is given we'll replace the mock response with the actual API.


Thank you @Siddhant-K-code for the shoutout, and thank you @jatingodnani for showing your interest, please join the discord [contribution](https://discord.com/invite/8hRaqhK) channel to get informed about new contribution opportunities.

**Comment by jatingodnani** [comment]
@Nishchit14 i think you have already assign another issue to @rohitmondal03

**Comment by Nishchit14** [comment]
@jatingodnani This one is a small one, I think He is quite near to finishing it so I assigned it to him.

But you can still work on it, If it's ready soonest I'll accept your PR.

**Comment by jatingodnani** [comment]
@rohitmondal03 are you working on this issue?

**Comment by rohitmondal03** [comment]
@Nishchit14 can you please givve "hacktoberfest accepted label" in my merge request. The request is showing "not-accepted" in my hacktoberfest profile !!🙏

**Comment by Nishchit14** [comment]
@rohitmondal03 it's done, check now.
